### DISCUSSION TITLE
Support SCIM for roles

### DISCUSSION THREAD

**Discussion by leahy268** [tinacms/tinacms#3816] [Ideas]
Hi Guys, 

I know you support SSO which is great
Do you have plans to support SCIM for importing users and assigning roles.  I think that would be really beneficial, especially for larger enterprises.

Please see my software suggestion here:

https://bettersoftwaresuggestions.com/tina/tina-cms-help-me-to-add-users-to-roles-automatically/

**Comment by scottgallant** [comment]
Thanks for the feedback, @leahy268. This is something that could work with Tina but would take a little bit of work to support.  It's not on the roadmap currently. I'll let you know if that changes.
